test/integration/test_spreadsheet_generator.py

```diff
@@ -12,7 +13,11 @@
 import tempfile
 import yaml
 
+
 class TestSpreadsheetGenerator(TestCase):
```

Comment:
This test file is very big. Can it be split to smaller logical units?

Reply:
I believe we could do that later. This test suite is not part of the user story. I just tried to fix them because they're failing.